appinfo/info.xml

```diff
@@ -15,8 +15,8 @@ In addition to managing synchronized contacts the application also offers a feat
     <author>Hendrik Leppelsack, Thomas Müller, Alexander Weidinger, Jan-Christoph Borchardt, Tom Needham</author>
     <version>1.5.5</version>
     <documentation>
-    	<user>https://doc.owncloud.org/server/10.0/user_manual/pim/index.html</user>
-    	<admin>https://doc.owncloud.org/server/10.0/admin_manual/configuration/server/occ_command.html?highlight=contacts#dav-commands</admin>
+    	<user>https://doc.owncloud.com/webui/next/classic_ui/pim/</user>
+    	<admin>https://doc.owncloud.com/server/next/admin_manual/configuration/server/occ_command.html</admin>
```

Comment:
Link should point to the DAV Commands section so it should rather be: https://doc.owncloud.com/server/next/admin_manual/configuration/server/occ_command.html#dav-commands